arch/arm64/boot/dts/qcom/sdm625-xiaomi-vince.dts

```diff
@@ -0,0 +1,332 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
@fekz115 @M0Rf30 Please provide your name and email for author name & sign off in the commit message

https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin

Reply:
Eugene Lepshy - fekz115@gmail.com

So, there is some fixmes in the file, but you didn't commented it. So, are you agree to send it, or you just missed?

Reply:
reserved-memory and wled I will try and handle

Reply:
Gianluca Boiano - morf3089@gmail.com